src/mesh/SX1262Interface.cpp

```diff
@@ -34,11 +34,11 @@ bool SX1262Interface::init()
     pinMode(SX1262_TXEN, OUTPUT);
 #endif
 
-#ifndef SX1262_E22
+#if !defined(SX1262_E22) && !defined(SX1262_USE_DIO3_FOR_TCXO)
```

Comment:
hmm - what's the intent of SX1262_USE_DIO3_FOR_TCXO?

Reply:
The intent is to tell the radio interface that it has TCXO connected to DIO3 -  same as in SX1262_E22. I could have defined SX1262_E22 in my variant.h, but since my board is not using E22, it felt wrong. Now I see that ppr1/variant.h is doing exactly that, I could be convinced to copy the existing approach :). LMK what you think.

Reply:
@vfurman-gh yes, would you mind doing that.  It seems like there are a number of E22 clones now (and I know of at least one more coming).   I think that would avoid confusion.  I'll merge these (thanks for fixing/generalizing the RTC code for nrf52 especially) changes now but if you could send in a separate PR to change that it would be appreciated.

Reply:
https://github.com/meshtastic/Meshtastic-device/pull/747